plugins/pencilblue/templates/angular/admin/content/objects/object_form.html

```diff
@@ -1,251 +1,210 @@
-            field.paginationIndex = paginationIndex;
+  $scope.getSortListeners = function(field) {
+    var active = $scope.customObject[field.name];
+    var inactive = field.available_objects;
 
-            paginationService.paginate(field.available_objects, field.paginationIndex, field.paginationLimit, function(availableObjects, paginationPages) {
-                field.available_objects = availableObjects;
-                field.paginationPages = paginationPages;
-            })
+    if(!active) {
+      active = [];
+    }
+
+    return {
+      accept: function (sourceItemHandleScope, destSortableScope) {
+        var itemToTest = sourceItemHandleScope.itemScope.object;
+
+        for(var i = 0; i < active.length; i++) {
+          if(itemToTest._id.toString() === active[i]._id.toString()) {
+            return true;
+          }
+        }
+        for(var i = 0; i < inactive.length; i++) {
+          if(itemToTest._id.toString() === inactive[i]._id.toString()) {
+            return true;
+          }
         }
 
-        $scope.pageButtonVisible = function(field, testIndex) {
-            return paginationService.pageButtonVisible(testIndex, field.paginationIndex, field.paginationPages);
-        };
-
-        $scope.getSortListeners = function(field) {
-            var active = $scope.customObject[field.name];
-            var inactive = field.available_objects;
-
-            if(!active) {
-                active = [];
-            }
-
-            return {
-                accept: function (sourceItemHandleScope, destSortableScope) {
-                    var itemToTest = sourceItemHandleScope.itemScope.object;
-
-                    for(var i = 0; i < active.length; i++) {
-                        if(itemToTest._id.toString() === active[i]._id.toString()) {
-                            return true;
-                        }
-                    }
-                    for(var i = 0; i < inactive.length; i++) {
-                        if(itemToTest._id.toString() === inactive[i]._id.toString()) {
-                            return true;
-                        }
-                    }
-
-                    return false;
-                },
-                itemMoved: function (event) {},
-                orderChanged: function(event) {},
-                containment: '#' + field.name
-            }
-        };
-
-        $scope.getFieldUID = function(field) {
-            return field.name.toLowerCase().split(' ').join();
+        return false;
+      },
+      itemMoved: function (event) {},
+      orderChanged: function(event) {},
+      containment: '#' + field.name
+    }
+  };
+
+  $scope.getFieldUID = function(field) {
+    return field.name.toLowerCase().split(' ').join();
+  }
+
+  $scope.setBoolean = function(fieldName, setting) {
+    $scope.customObject[fieldName] = setting;
+  };
+
+  $scope.setDateToNow = function(fieldName) {
+    $scope.customObject[fieldName] = $filter('date')(new Date(), 'MM-dd-yyyy HH:mm');
```

Comment:
We could make it it's own directive and bring that in, or we could just
make it a variable outside of angular and bring it into scope. This is so
small that I think the first approach would be overkill.

On Mon, Jun 22, 2015 at 9:48 AM, Brian <notifications@github.com> wrote:

> In
> plugins/pencilblue/templates/angular/admin/content/objects/object_form.html
> <https://github.com/pencilblue/pencilblue/pull/660#discussion_r32935160>:
>
> > +      itemMoved: function (event) {},
> > +      orderChanged: function(event) {},
> > +      containment: '#' + field.name
> > +    }
> > +  };
> > +
> > +  $scope.getFieldUID = function(field) {
> > +    return field.name.toLowerCase().split(' ').join();
> > +  }
> > +
> > +  $scope.setBoolean = function(fieldName, setting) {
> > +    $scope.customObject[fieldName] = setting;
> > +  };
> > +
> > +  $scope.setDateToNow = function(fieldName) {
> > +    $scope.customObject[fieldName] = $filter('date')(new Date(), 'MM-dd-yyyy HH:mm');
>
> For the angular noob, what kind of effort does that entail? I was thinking
> that was just injection of a built in service that just had to be
> configured to point to the localization files. I'm probably over
> simplifying it though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pencilblue/pencilblue/pull/660/files#r32935160>.
>